docs/section22/Rule22-7.md

```diff
@@ -0,0 +1,48 @@
+
+# CHW&CW - Rule 22-7  
+
+**Rule ID:** 22-7  
+**Rule Description:** Baseline chilled water system that does not use purchased chilled water shall be modeled as primary/secondary systems.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.10 Chilled-water supply temperature reset (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13.
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13: `PLACEHOLDER`
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water: `if B_RMR.ASHRAE229.chillers:`
+
+## Rule Logic:  
+
```

Comment:
I would update the rule logic to confirm that 
(a) there is a loop referenced by a chiller (this would be the primary loop), 
(b) that this primary loop references a child loop (this would be the secondary loop), 
(c) that the secondary loop is not referenced by another chiller or boiler and does not have a child loop of its own (tetriary loop)
(d) that there are no loops of type "cooling" other than these two.

Reply:
Do we also need to check that all HVAC cooling coils are on the secondary loop and not on primary loop.